pkg/inject/sidecar_test.go

```diff
@@ -523,8 +525,8 @@ func TestSidecarExplicitTags(t *testing.T) {
 
 	// verify
 	assert.Len(t, dep.Spec.Template.Spec.Containers, 2)
-	agentTags := agentTags(dep.Spec.Template.Spec.Containers[1].Args)
-	assert.Equal(t, []string{"key=val"}, agentTags)
+	agentTags := parseAgentTags(dep.Spec.Template.Spec.Containers[1].Args)
+	assert.True(t, reflect.DeepEqual(agentTags, map[string]string{"key": "val"}))
```

Comment:
Why can't this be `assert.Equal` as it was before?

Reply:
Changed